### PR TITLE
Added AnimatedVisibilityExample.kt to MotionCompose

### PR DESCRIPTION
**Pull request by riggaroo**
Added AnimatedVisibility Sample to MotionCompose to demonstrate Animation Preview tooling. 

https://user-images.githubusercontent.com/9973046/175513421-1e804730-e909-4956-9c36-9f0899670fc4.mov

### CODE REVIEW

File: MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/visibility/AnimatedVisibilityExample.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package com.example.android.compose.motion.demo.visibility
```
Comment (yaraki) [inline MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/visibility/AnimatedVisibilityExample.kt:1]: License header

**Review by thagikura — COMMENTED** [review]
Thanks Rebecca! LGTM other than Yuichi's comment.

File: MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/visibility/AnimatedVisibilityExample.kt (hunk 1 above)
Comment (riggaroo) [reply MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/visibility/AnimatedVisibilityExample.kt:16]: done

**Review by thagikura — APPROVED** [review]
LGTM!

File: MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/visibility/AnimatedVisibilityExample.kt (hunk 1 above)
Comment (Osoted56) [reply MotionCompose/app/src/main/java/com/example/android/compose/motion/demo/visibility/AnimatedVisibilityExample.kt:16]: Está clonada mi cuenta

**Pull request merged**